graphql_api/types/invoice/invoice.py

```diff
@@ -0,0 +1,129 @@
+from functools import cached_property
+from typing import List, Optional
+
+from ariadne import ObjectType
+from stripe import (
+    Invoice,
+    InvoiceLineItem,
+    ListObject,
+    PaymentMethod,
+)
+
+invoice_bindable = ObjectType("Invoice")
+
+
+@invoice_bindable.field("id")
+def resolve_invoice_id(invoice: Invoice, info) -> str | None:
+    return invoice["id"]
+
+
+@invoice_bindable.field("number")
+def resolve_invoice_number(invoice: Invoice, info) -> str | None:
+    return invoice["number"]
+
+
+@invoice_bindable.field("status")
+def resolve_invoice_status(invoice: Invoice, info) -> str | None:
+    return invoice["status"]
+
+
+@invoice_bindable.field("created")
+def resolve_invoice_created(invoice: Invoice, info) -> int:
+    return invoice["created"]
+
+
+@invoice_bindable.field("periodStart")
+def resolve_invoice_period_start(invoice: Invoice, info) -> int:
+    return invoice["period_start"]
+
+
+@invoice_bindable.field("periodEnd")
+def resolve_invoice_period_end(invoice: Invoice, info) -> int:
+    return invoice["period_end"]
+
+
+@invoice_bindable.field("dueDate")
+def resolve_invoice_due_date(invoice: Invoice, info) -> int | None:
+    return invoice["due_date"]
+
+
+@invoice_bindable.field("customerName")
+def resolve_invoice_customer_name(invoice: Invoice, info) -> str | None:
+    return invoice["customer_name"]
+
+
+# NOTE: This doesn't currently look to be used in gazebo, maybe can remove?
```

Comment:
Fine to remove these unused ones imo.

Reply:
Gonna do all the gut removal stuff in a followup PR if that's aight